### PR TITLE
chore: update readme

### PR DESCRIPTION
**Pull request by alashuk-applicaster**
## Description
Update README

[Monday ticket](https://applicaster.monday.com/boards/1615228522/views/35072856/pulses/4308708015?userId=17198506)

`Please describe PR here`

<details>
  <summary>Platforms Affected</summary>
  
- [ ] Android
- [ ] Android TV
- [ ] Apple TV (TVOS)
- [ ] Fire TV (FireOS)
- [ ] iOS
- [ ] LG TV (WebOS)
- [ ] Samsung TV (Tizen)
- [ ] Web
</details>

## Known or Pending Issues

`fill in this section with potential issues / limitations the reviewer(s) should know about`

## Screenshots

`If there is some kind of ui change please share screenshots`

## Version Info

`Does this update require a certain version of quickbrick, or does it rely on another pr to work?`

### CODE REVIEW

File: plugins/quick-brick-matchbox/README.md [hunk 1]
```diff
@@ -55,8 +54,6 @@ Tested on Android mobile and ios Mobile.
         "gameNumber": "1",
         "TimeOfGame": "15:00",
         "PlaceOfGameHeb": "היכל קרפנטייה",
```
Comment (snooky23) [inline plugins/quick-brick-matchbox/README.md:56]: Let's change this key to `gameLocation`. On the datasrouce, lets create create the game location value and keep the old one for now.

File: plugins/quick-brick-matchbox/README.md [hunk 2]
```diff
@@ -55,8 +54,6 @@ Tested on Android mobile and ios Mobile.
         "gameNumber": "1",
         "TimeOfGame": "15:00",
```
Comment (snooky23) [inline plugins/quick-brick-matchbox/README.md:55]: first letters should be lower case. please adjust for all.

File: plugins/quick-brick-matchbox/README.md [hunk 3]
```diff
@@ -55,8 +54,6 @@ Tested on Android mobile and ios Mobile.
...
         "PlaceOfGameHeb": "היכל קרפנטייה",
-        "tickets_url": "",
-        "ExternalID": "0",
```
Comment (snooky23) [inline plugins/quick-brick-matchbox/README.md:59]: What is match status stands for?

File: plugins/quick-brick-matchbox/README.md [hunk 4]
```diff
@@ -55,8 +54,6 @@ Tested on Android mobile and ios Mobile.
...
         "matchStatus": 1,
         "button1_params": {
             "isEnabled": false,
```
Comment (snooky23) [inline plugins/quick-brick-matchbox/README.md:59]: Please remove the isEnabled parameters

File: plugins/quick-brick-matchbox/README.md [hunk 5]
```diff
@@ -80,17 +74,15 @@ Tested on Android mobile and ios Mobile.
...
 ```
 
 ## Mandatory params for match box
 
-Inside each entry in ``extensions``: 
+Inside each entry in ``extensions``:
+
     1. `team1_name` / `team2_name`;
     2. `team1_pic` / `team2_pic`;
     3. `TimeOfGame`;
```
Comment (snooky23) [inline plugins/quick-brick-matchbox/README.md:88]: should be lower case

**Review by snooky23 — CHANGES_REQUESTED** [review]

File: plugins/quick-brick-matchbox/README.md (hunk 2 above)
Comment (alashuk-applicaster) [reply plugins/quick-brick-matchbox/README.md]: done

File: plugins/quick-brick-matchbox/README.md (hunk 3 above)
Comment (alashuk-applicaster) [reply plugins/quick-brick-matchbox/README.md:59]: Depends on match status we can see different types of content (past/live/future)

File: plugins/quick-brick-matchbox/README.md (hunk 4 above)
Comment (alashuk-applicaster) [reply plugins/quick-brick-matchbox/README.md]: done

File: plugins/quick-brick-matchbox/README.md (hunk 5 above)
Comment (alashuk-applicaster) [reply plugins/quick-brick-matchbox/README.md]: done

File: plugins/quick-brick-matchbox/README.md (hunk 3 above)
Comment (snooky23) [reply plugins/quick-brick-matchbox/README.md:59]: @alashuk-applicaster can you please add this also to the readme? please write all the possible values we can use.

**Review by snooky23 — CHANGES_REQUESTED** [review]

File: plugins/quick-brick-matchbox/README.md (hunk 3 above)
Comment (alashuk-applicaster) [reply plugins/quick-brick-matchbox/README.md:59]: done

**Review by snooky23 — APPROVED** [review]

**Pull request closed without merging**